### PR TITLE
Adds the Cluwnescape mutation to the game.

### PR DESCRIPTION
**Pull request by Iamgoofball**
oqvja9e
buying_gf
#8888

Adminspawn/telecomms fuckery only.

http://www.dafont.com/runescape-uf.font

used font is PUBLIC DOMAIN so we're GOOD2GO

### CODE REVIEW

**Comment by GunHog** [comment]
We. Are. So. Boned. :+1:

**Comment by SconesC** [comment]
Quality genetics addition, does not in any way only further the primary issue that is stopping the job from being playable

**Comment by Iamgoofball** [comment]
@SconesC actually, I didn't add an extra block for this one, so that problem isn't being increased.

I'll probably knock the block amounts down soon.

**Comment by Incoming5643** [comment]
can it only replace other disabilities, or can it replace a power?

If it can't replace powers, holy fuck dial back the number of blocks yesterday.

**Comment by Cheridan** [comment]
lay off with the stupid gimmick fonts

**Comment by Incoming5643** [comment]
slayer

File: code/datums/mutations.dm [hunk 1]
```diff
@@ -1,4 +1,6 @@
 /var/global/list/mutations_list = list()
+var/thanks_tobba = 'icons/fonts/runescape_uf.ttf'
```
Comment (Miauw62) [inline code/datums/mutations.dm]: >thanks_tobba

Comment (Iamgoofball) [inline code/datums/mutations.dm]: Tobba helped me get custom fonts working.

**Pull request closed without merging**